Python/Product/Pylance/source.extension.vsixmanifest

```diff
@@ -45,9 +45,5 @@
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="File" Path="Microsoft.PythonTools.Ipc.Json.dll" />
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="File" Path="Microsoft.PythonTools.Attacher.exe" />
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="File" Path="Microsoft.PythonTools.ProjectWizards.dll" />
-    <Asset Type="Microsoft.VisualStudio.Package" d:Source="Project" d:ProjectName="Cookiecutter" />
-    <Asset Type="Microsoft.VisualStudio.Package" d:Source="Project" d:ProjectName="Profiling" />
-    <Asset Type="Microsoft.VisualStudio.ProjectTemplate" Path="ProjectTemplates" />
-    <Asset Type="Microsoft.VisualStudio.ItemTemplate" Path="ItemTemplates" />
```

Comment:
this change intentional ?

Reply:
Yes as those are going away. Well we hope anyway.